packages/node_modules/@ciscospark/internal-plugin-presence/test/integration/spec/presence.js

```diff
@@ -64,6 +74,8 @@ describe(`plugin-presence`, function() {
     });
 
     describe(`#subscribe`, () => {
+      afterEach(() => spock.spark.internal.presence.unsubscribe([mccoy.id, spock.id]));
```

Comment:
`spock && spock.spark` makes explaining jenkins output a lot easier when test user creation fails

Reply:
I'm not sure I follow this.